FlowJo/FlowJo.munki.recipe

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Description</key>
+	<string>Downloads FlowJo and imports into munki.
+</string>
+	<key>Identifier</key>
+	<string>com.github.its-unibas.munki.FlowJo</string>
+	<key>Input</key>
+	<dict>
+		<key>MUNKI_REPO_SUBDIR</key>
+		<string>apps/%NAME%</string>
+		<key>NAME</key>
+		<string>FlowJo</string>
+		<key>pkginfo</key>
+		<dict>
+			<key>catalogs</key>
+			<array>
+				<string>testing</string>
+			</array>
+			<key>category</key>
+			<string>Science</string>
+			<key>description</key>
+			<string>The FlowJo® workspace is a powerful statistical environment that is used for immunophenotyping, cell cycle, proliferation, kinetics studies, quantitative population comparison, or plate screening assays to name a few.</string>
+			<key>developer</key>
+			<string>FlowJo</string>
+			<key>display_name</key>
+			<string>FlowJo</string>
+			<key>minimum_os_version</key>
+			<string>10.7</string>
+			<key>name</key>
+			<string>%NAME%</string>
+			<key>unattended_install</key>
+			<true/>
+			<key>items_to_copy</key>
+	    <array>
+	        <dict>
+			<key>destination_path</key>
+			<string>/Applications</string>
+			<key>source_item</key>
+			<string>FlowJo.app</string>
+		 </dict>
+		 <dict>
+			<key>destination_path</key>
+			<string>/Applications/FlowJo.app/Contents/Resources</string>
+			<key>source_item</key>
+			<string>plugins</string>
+		 </dict>
+	    </array>
+			<key>postinstall_script</key>
+			<string>#!/bin/bash
```

Comment:
Could you add a short comment in the postinstall script as to why it's necessary?
This might help people who are not familiar with the application.

Reply:
Thanks!
Sure, did that in my last commit.